baselines/dom.generated.d.ts

```diff
@@ -18736,6 +18736,8 @@ declare var Response: {
     new(body?: BodyInit | null, init?: ResponseInit): Response;
     /** [MDN Reference](https://developer.mozilla.org/docs/Web/API/Response/error_static) */
     error(): Response;
+    /** [MDN Reference](https://developer.mozilla.org/docs/Web/API/Response/json) */
```

Comment:
The https://developer.mozilla.org/docs/Web/API/Response/json page describes the `Response.prototype.json()` method. The `static Response.json(data, init)` method isn't yet documented in MDN. https://github.com/mdn/content/issues/21550


The URL will probably be: `https://developer.mozilla.org/docs/Web/API/Response/json_static`

Reply:
Hello and thanks for the review. 
You're absolutely right, the right URL does not exist yet.
Unfortunately, using it (or removing the comment) fails the test suite.

Reply:
`https://developer.mozilla.org/docs/Web/API/Response/json_static` has been created last week.